python-lib/dku_deeplearning_image/utils.py

```diff
@@ -126,7 +126,9 @@ def get_file_path(folder_path, file_name):
 
 
 def get_cached_file_from_folder(folder, file_path):
-    filename = file_path.replace('/', '_') if isinstance(file_path, str) else file_path.replace(b'/', b'_')
+    if isinstance(file_path, list):
+        file_path = file_path[0]
+    filename = file_path.replace('/', '_') if isinstance(file_path, str) else file_path[0].replace(b'/', b'_')
```

Comment:
I think it should be the opposite, because this code would fail for unicode in py2 for instance, so check whether its `bytes`, else consider it as a string (would also work for unicode)

Reply:
Fixed in c19a7e2. Indeed, I simplified the syntax of it to be easier to read